src/pages/patientView/PatientViewPage.tsx

```diff
@@ -79,8 +79,19 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     {
                         patientViewPageStore.setSampleId(query.sampleId as string);
                     }
-                    patientViewPageStore.patientIdsInCohort = ('navCaseIds' in query ? (query.navCaseIds as string).split(",") : []);
 
+                    let patientIdsInCohort = ('navCaseIds' in query ? (query.navCaseIds as string).split(",") : []);
+                    
+                    let multiStudyCohorts: string[] = patientIdsInCohort
+                    .filter(entity=> entity.split(':').length == 2);
```

Comment:
why not just do entity.contains(":").  no need to split since you're not saving the result

Reply:
actually, see comment below. maybe not necessary to determine if we have multiStudyCohort?